app/Http/Controllers/TaskController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Controllers;
+use \App\Task;
+use Session;
+
+
+use Illuminate\Http\Request;
+
+class TaskController extends Controller
+{
+    public function fetchUser()
+    {
+        return Session::get('username');
+    }
+
+    public function fetchTasks() {
+       return Task::where('created_by', $this->fetchUser())->get();
+    }
```

Comment:
Both of these functions are only used by this class. They should be `private` functions, not public. This likely won't affect anything you're doing, but it could cost you on a job interview.